src/System.Net.NameResolution/src/System/Net/Dns.cs

```diff
@@ -88,45 +105,50 @@ public static Task<IPHostEntry> GetHostEntryAsync(IPAddress address)
 
             if (address.Equals(IPAddress.Any) || address.Equals(IPAddress.IPv6Any))
             {
+                if (NetEventSource.IsEnabled) NetEventSource.Error(address, $"Invalid address {address}");
```

Comment:
I think this should be `NetEventSource.Error(address, $"Invalid address '{address}'");`

Using single apostrophe marks will help readability. There are few places in these files that could benefit.

This is similar to what we do in exception message text, see https://github.com/dotnet/corefx/pull/42229#discussion_r341878219